packages/block-library/src/navigation/index.php

```diff
@@ -1415,7 +1415,7 @@ function block_core_navigation_update_ignore_hooked_blocks_meta( $post ) {
 
 // Injection of hooked blocks into the Navigation block relies on some functions present in WP >= 6.4
 // that are not present in Gutenberg's WP 6.4 compatibility layer.
```

Comment:
The comment is no longer accurate 😬 We should change it to something like

```suggestion
// Injection of hooked blocks into the Navigation block relies on some functionality present in WP >= 6.5
// that are not present in Gutenberg's WP 6.5 compatibility layer.
```